src/test/java/de/mediathekview/mserver/crawler/wdr/parser/WdrVideoJsonDeserializerTest.java

```diff
@@ -0,0 +1,95 @@
+package de.mediathekview.mserver.crawler.wdr.parser;
+
+import com.google.gson.JsonElement;
+import de.mediathekview.mserver.crawler.wdr.WdrMediaDTO;
+import de.mediathekview.mserver.testhelper.JsonFileReader;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Optional;
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.CoreMatchers.notNullValue;
+import static org.junit.Assert.assertThat;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+
+@RunWith(Parameterized.class)
+public class WdrVideoJsonDeserializerTest {
+  
+  @Parameterized.Parameters
+  public static Collection<Object[]> data() {
+    return Arrays.asList(new Object[][] { 
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this array creation and simply pass the elements. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3878)